tests/Functional/Schema/PostgreSQLSchemaManagerTest.php

```diff
@@ -484,6 +486,67 @@ public function testAlterTableAutoIncrementIntToBigInt(string $from, string $to,
         self::assertTrue($tableFinal->getColumn('id')->getAutoincrement());
     }
 
+    public function testListTableColumnsOidConflictWithNonTableObject(): void
+    {
+        if (version_compare($this->connection->getServerVersion(), '12.0', '<')) {
+            self::markTestSkipped('Manually setting the Oid is not supported in Postgres 11 and earlier');
+        }
```

Comment:
I had to adjust the version constraint here while merging up. Backported as #5955.